pkg/csi/service/wcpguest/controller.go

```diff
@@ -199,13 +237,24 @@ func (c *controller) ReloadConfiguration() {
 func (c *controller) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest) (
 	*csi.CreateVolumeResponse, error) {
 
+	start := time.Now()
+	isFileVolumeRequest := common.IsFileVolumeRequest(ctx, req.GetVolumeCapabilities())
```

Comment:
`isFileVolumeRequest` can be used at Line: 311, instead of calling `common.IsFileVolumeRequest(ctx, req.GetVolumeCapabilities())` again

Reply:
Done.